pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, gcc11Stdenv
+, fetchFromGitHub
+, cmake
+, ninja
+, unzip
+, wrapQtAppsHook
+, makeWrapper
+, qtbase
+, qttools
+}:
+
+let serenity = fetchFromGitHub {
+  owner = "awesomekling";
```

Comment:
The canonical repository URL for serenity is under the SerenityOS organization

Reply:
oops. fixed, thanks!